backup-postgres/action.yml

```diff
@@ -23,6 +23,12 @@ inputs:
   backup-file:
     description: Name of the backup file
     required: true
+  keyvault-name:
+    description: Name of the key vault containing the slack webhook
+    required: true
+  backup-env:
```

Comment:
This is only used in the summary. I think it can be removed as we already have the app name and DB server name